test/CreatorToken.t.sol

```diff
@@ -49,6 +49,44 @@ contract CreatorTokenTest is Test {
     (uint256 _creatorFee, uint256 _adminFee) = creatorToken.calculateFees(_price);
     return _price + _creatorFee + _adminFee;
   }
+
+  function buyAToken(address _buyer) public {
+    (uint256 _tokenPrice, uint256 _creatorFee, uint256 _adminFee) = creatorToken.nextBuyPrice();
+    uint256 _totalPrice = _tokenPrice + _creatorFee + _adminFee;
+    uint256 originalCreatorTokenBalance = payToken.balanceOf(address(creatorToken));
+    uint256 originalBuyerBalance = creatorToken.balanceOf(_buyer);
```

Comment:
A few thoughts here:

* We dropped the convention of using `_` for non-storage vars. These should respect that for clarity.
* One of these is the balance of the paytoken (i.e. how much $PAY) the other is the balance of the creator token, i.e. how much $TEST. Our variable names should be more verbose to make it clear what's going on. This is a test so we shouldn't worry much about being "too" verbose. For example something like `_originalPayTokenBalanceOfCreatorTokenContract` and `_originalBuyerBalanceOfCreatorTokens` both seem fair game to me.
* Why *don't* we have a variable to track the original $PAY balance of the buyer, and assertion at the end of the test that it has changed by the right amount? Seems like we'd want that too.

Reply:
Makes sense, updating naming in the same commit as above: https://github.com/showtime-xyz/creator-tokens/pull/14/commits/89f1666530442849e5fcc6a35261447ee2cf1cbf

Reply:
>     * Why _don't_ we have a variable to track the original $PAY balance of the buyer, and assertion at the end of the test that it has changed by the right amount? Seems like we'd want that too.

Yes, we do! Added that check here: https://github.com/showtime-xyz/creator-tokens/pull/14/commits/6a0ad6858e103fa6da6384492f59571f85783ba2